index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8"/>
+    <title>Renoma</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+    <header>
+        <nav>
+            <div class="Hamburger"><a><img src="images/Hamburger.png" alt="Hamburger butten image"></a></div>
+            <div class="Vertical-divider"></div>
```

Comment:
Change please class name in lower case `<div class="vertical-divider">`
**Note**:
All class names should be in the lower case